source/overview/product.rst

```diff
@@ -22,47 +22,50 @@ Core committers, including both community contributors and paid staff at Matterm
 Mattermost Editions
 -----------------------
 
-Mattermost is offered in three editions:
+Mattermost is offered in three editions, in multiple languages including U.S. English, Chinese (Simplified and Traditional), Dutch, French, German, Italian, Japanese, Korean, Polish, Brazilian Portuguese, Romanian, Russian, Turkish, Spanish, and Ukrainian.
```

Comment:
I think this fits better here. In a similar vein, I propose adding MFA to this introduction as it's applied across all versions. 3/5.

Reply:
@justinegeffen All of the Team Edition features apply across all versions. Perhaps we should make that more clear -- e.g. for E10 we say `Building on the features in Team Edition, it offers the following advanced features:`

Reply:
Agreed. I've given my MFA suggestion more thought. Adding your suggestion would accomplish the same goal as my more vague one so I'm going to go with that.

Reply:
I've added the link to the final paragraph of the document.